modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/components/SpaceSticker.tsx

```diff
@@ -6,44 +6,50 @@
 import ClaySticker from '@clayui/sticker';
 import React from 'react';
 
-function getDisplayType(
-	char: string
-): React.ComponentProps<typeof ClaySticker>['displayType'] {
-	const validDisplayTypes: React.ComponentProps<
-		typeof ClaySticker
-	>['displayType'][] = [
-		'outline-0',
-		'outline-1',
-		'outline-2',
-		'outline-3',
-		'outline-4',
-		'outline-5',
-		'outline-6',
-		'outline-7',
-		'outline-8',
-		'outline-9',
-	];
+export type LogoColor = React.ComponentProps<typeof ClaySticker>['displayType'];
 
-	return validDisplayTypes[char.charCodeAt(0) % validDisplayTypes.length];
+export const logoColors: LogoColor[] = [
+	'outline-0',
+	'outline-1',
+	'outline-2',
+	'outline-3',
+	'outline-4',
+	'outline-5',
+	'outline-6',
+	'outline-7',
+	'outline-8',
+	'outline-9',
+];
+
+function getDisplayType(char: string): LogoColor {
+	return logoColors[char.charCodeAt(0) % logoColors.length];
 }
 
 export default function SpaceSticker({
+	className,
 	displayType,
+	hiddenName,
+	id,
 	name,
 	size,
 }: {
+	className?: string;
+	hiddenName?: boolean;
+	id?: string;
```

Comment:
This is open to debate: what if we just allow to pass `...otherProps` to this component, so we don't explicitly add the props `className` and `id`?